client/a8c-for-agencies/sections/marketplace/wpcom-overview/lib/wpcom-bulk-options.ts

```diff
@@ -0,0 +1,38 @@
+const wpcomBulkOptions = [
```

Comment:
Can we add a FIXME stating that we will fetch the discount bracket from a proper source instead of hardcoding it?